Purge "will" from Client Cfg (#663)
uyuni-project/uyuni-docs: modules/client-configuration/pages/snippets/trust_gpg.adoc

By default, some operating systems do not trust the GPG key for the {productname} client tools.
The clients can be successfully bootstrapped without the GPG key being trusted.

> However, can not install new client tool packages or update them until the keys are trusted.

Reviewer: ```suggestion
However, you cannot install new client tool packages or update them until the keys are trusted.
```
